tech/database/mariadb/about.md

```diff
@@ -42,16 +44,16 @@ If you need to connect to the MariaDB server using GUI, install either phpMyAdmi
 $ sudo dnf install phpMyAdmin
 ```
 
-Or install Libre Office Base with JDBC plug-in for MySQL:
+Or install Libre Office Base with JDBC plug-in for MariaDB:
 
 ```
-$ sudo dnf install libreoffice-base mysql-connector-java
+$ sudo dnf install libreoffice-base mariadb-java-client
 ```
 
-For connecting to the MariaDB server using ODBC, install `unixODBC` and `mysql-connector-odbc` packages:
+For connecting to the MariaDB server using ODBC, install `unixODBC` and `mariadb-connector-odbc` packages:
 
 ```
-$ sudo dnf install unixODBC mysql-connector-java
+$ sudo dnf install unixODBC mariadb-connector-odbc
 ```
 
 ## <a name="basic-tutorial-for-mariadb-in-fedora"></a>Basic tutorial for MariaDB in Fedora
```

Comment:
Ah, this is really ugly. Could you fix it?

Reply:
Not sure what I should to with it.
I don't have any good english article on this matter.
(I have an older in czech, though: https://mojefedora.cz/stitek/mariadb/ )

It may be better to drop that line entirely.

Reply:
It's always better to have less, but accurate content.
But in the first place what I've meant is the use of html (<a ...>) needlessly.